app/views/admin_page.erb

```diff
@@ -0,0 +1,15 @@
+<h1> ADMIN PAGE </h1>
+
+<a href="http://localhost:9292/gender_form">Gender Form</a>
```

Comment:
Link to just `/gender_form` instead of the full URL. That way it'll still work if we (for some reason) need to run the server on a port other than 9292.